base_domain_inverse_function/tests/test_partner_domains.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo.osv.expression import AND, OR
+from odoo.tests import TransactionCase
+
+from ..inverse_expression import inverse_AND, inverse_OR
+
+
+class TestPartnerDomains(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.partner_model = cls.env["res.partner"]
+        cls.partner_domains = [
+            [("name", "ilike", "Deco")],
```

Comment:
When odoo native module `crm` is installed, it changes email of the partner filtered by the `name` field. Thus test was failed as no partner could be found.
```suggestion
            [("display_name", "ilike", "Deco")],
```